src/@lekoarts/gatsby-theme-minimal-blog/components/tag.tsx

```diff
@@ -1,53 +1,71 @@
 /** @jsx jsx */
-import { jsx, Heading, Flex } from "theme-ui"
-import { HeadFC, Link } from "gatsby"
-import Layout from "./layout"
-import useMinimalBlogConfig from "../hooks/use-minimal-blog-config"
-import Listing from "./listing"
-import replaceSlashes from "../utils/replaceSlashes"
-import Seo from "./seo"
+import { jsx, Heading, Flex } from 'theme-ui';
+import { HeadFC, Link } from 'gatsby';
+import Layout from './layout';
+import useMinimalBlogConfig from '../hooks/use-minimal-blog-config';
+import Listing from './listing';
+import replaceSlashes from '../utils/replaceSlashes';
+import Seo from './seo';
 
 export type MBTagProps = {
-  posts: {
-    slug: string
-    title: string
-    date: string
-    excerpt: string
-    description: string
-    timeToRead?: number
-    tags: {
-      name: string
-      slug: string
-    }[]
-  }[]
+  data: {
+    allPost: {
+      nodes: {
+        slug: string;
+        title: string;
+        date: string;
+        excerpt: string;
+        description: string;
+        timeToRead?: number;
+        tags: {
+          name: string;
+          slug: string;
+        }[];
+        ns?: string;
+      }[];
+    };
+  };
   pageContext: {
-    isCreatedByStatefulCreatePages: boolean
-    slug: string
-    name: string
-  }
-}
+    isCreatedByStatefulCreatePages: boolean;
+    slug: string;
+    name: string;
+  };
+};
 
-const Tag = ({ posts, pageContext }: MBTagProps) => {
-  const { tagsPath, basePath } = useMinimalBlogConfig()
+const Tag = ({ data, pageContext }: MBTagProps) => {
+  const { tagsPath, basePath } = useMinimalBlogConfig();
+  const posts = data?.allPost?.nodes;
 
   return (
     <Layout>
-      <Flex sx={{ alignItems: `center`, justifyContent: `space-between`, flexFlow: `wrap` }}>
+      <Flex
+        sx={{
+          alignItems: `center`,
+          justifyContent: `space-between`,
+          flexFlow: `wrap`,
+        }}
+      >
         <Heading as="h1" variant="styles.h1" sx={{ marginY: 2 }}>
           {pageContext.name}
```

Comment:
This one should return translated version of the tag as a h1
![Screenshot by Dropbox Capture](https://github.com/romanr111/romanrud-blog/assets/12322543/5dfe659b-99d3-44dd-aa0f-4a471f111b3a)